### PR TITLE
Tidying the stylesheets

### PR DESCRIPTION
**Pull request by ahstro**
...and some related stuff.

I'm not even sure what I was doing anymore, but I stumbled upon an unused CSS selector, decided to look for more, and found quite a few. I'm honsetly not sure if these are somehow used through some sort of magic, but I searched through the entire codebase and found no other references to them, so they should be safe to remove.

Other than remove unused selectors, I organized some of them, fixed the stylization of a few, replaced one, as well as replaced the ubiqitous `#eee` with the already existing `@gray-lighter` variable.

This should hopefully clear the CSS/LESS up a bit, but don't just trust me, get in there and read (specifically the `next-challenge-button` and `delay-half` parts, they confused me) ; I did my best to make sure I'm not deleting some classes, essential to FCC's design, but I'm not making any promises.

### CODE REVIEW

**Comment by particlesurfer** [comment]
link?

File: public/css/main.less [hunk 1]
```diff
@@ -48,7 +47,7 @@ input[type=checkbox]
 }
 
 body.full-screen-body-background {
-  background-color: #eeeeee;
+  background-color: @gray-lighter;
 }
```
Comment (BerkeleyTrue) [inline public/css/main.less]: These are not the same.

Comment (ahstro) [inline public/css/main.less]: Why not? According to [line 15 in Bootstrap's variables.less](https://github.com/FreeCodeCamp/freecodecamp/blob/staging/public/css/lib/bootstrap/variables.less#L15), @gray-lighter is #eee, and I've been using it in all the other places where #eee is being used.

**Comment by QuincyLarson** [comment]
Just a heads-up, @ahstro, we plan to review all this when time permits. Thanks for your patience.

**Comment by ahstro** [comment]
No problem, take your time :) I understand you've got quite a lot on your hands right now, feels like every time I hitup FCC/map, another 1000 people have joined.

**Comment by QuincyLarson** [comment]
It took a while to get to this, but it looks great! I think you eliminated a lot of irrelevant code and fixed some Less variable assignment to make things more robust. Thanks for your help. Merged!

**Comment by bugron** [comment]
@QuincyLarson [L98](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/server/views/coursewares/showHTML.jade#L98), [L89](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/server/views/coursewares/showHTML.jade#L89), [L102](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/server/views/coursewares/showHTML.jade#L102) ?

**Comment by BerkeleyTrue** [comment]
@bugron What are you point out?

**Comment by bugron** [comment]
@BerkeleyTrue nothing anymore. PR was merged with conflicts which are now eliminated.

**Pull request closed without merging**